doc/ref_impl/cntt-ri2/chapters/chapter04.md

```diff
@@ -169,14 +144,16 @@ isolcpus: <String of cores to isolate>
   # String of cores/threads to isolate, e.g. "4-8,12"
 sst_bf_configuration_enabled: false
   # Speed Select Technology - Base Frequency (SST-BF). Can be enabled if processor is supported.
-```
```

Comment:
I think we still need to end the codeblock here to avoid formatting issues further on

Reply:
Done